runtime/ops/os/mod.rs

```diff
@@ -87,7 +85,7 @@ fn op_set_env(
   key: String,
   value: String,
 ) -> Result<(), AnyError> {
-  state.borrow_mut::<Permissions>().env.check(&key)?;
+  state.borrow_mut::<PermissionsContainer>().check_env(&key)?;
```

Comment:
I guess these don't really need to be borrow_mut anymore.

Reply:
Ah true. I'll change that to regular borrows

Reply:
Actually I will leave it as is - traits from various `ext/` crates require mutable reference (because they don't know that we're storing this behind `Arc<Mutex<>>` and it feels better to have consistent signature (also suggests that it can be internally mutated).